src/main/java/test/gc/switchele/commands/Element.java

```diff
@@ -1,24 +1,32 @@
 package test.gc.switchele.commands;
 
 import emu.grasscutter.GameConstants;
+import test.gc.switchele.LanguageHelper;
 
 public enum Element {
-    elementless(501, 701),
-    pyro(502, 702),
-    hydro(503, 703),
-    anemo(504, 704),
-    cryo(505, 705),
-    geo(506, 706),
-    electro(507, 707),
-    dendro(708, 708);
+    elementless(1),
+    pyro(2),
+    hydro(3),
+    anemo(4),
+    cryo(5),
+    geo(6),
+    electro(7),
+    dendro(8);
     private final int boyId;
     private final int girlId;
-
-    Element(int boyId, int girlId) {
-        this.boyId = boyId;
-        this.girlId = girlId;
+    private final int ElementId;
+    public int plv=0;
+    Element(int elementId) {
+        this.ElementId=elementId;
+        this.boyId = 500+ElementId;

```

Comment:
the Male Dendro traveller use 508,but it can not use in my  2.8.52
and Female Dendro traveller use 708,it can work normally
I think it is because my gc resources isnot newest

Reply:
For now Dendro is meant for peoples with 2.8.5 beta so it's better to keep male traveller at 708. If Mhy change this to 508 then I'll also update it, but for now I have to keep it as 708